app/pdf_redactor.py

```diff
@@ -601,16 +603,14 @@ def update_text_layer(options, text_tokens, page_tokens):
 		text_content = "".join(t.value for t in text_tokens)
 		matches = [a for a in pattern.finditer(text_content)]
 		if not matches:
-			current_app.logger.warning(
-                "No matches for address block during redaction procedure"
-            )
-			return
+			message = "No matches for address block during redaction procedure"
+			current_app.logger.warning(message)
+			return message
 
 		if len(matches) > 1:
-			current_app.logger.warning(
-                "More than one match for address block during redaction procedure"
-            )
-			return
+			message = "More than one match for address block during redaction procedure"
+			current_app.logger.warning(message)
+			return message
```

Comment:
we should add `return None` to the end of this function (line 667) just to be explicit